tests/microbenchmarks/bench_moe/timing/nsys.py

```
@@ -0,0 +1,89 @@
+
+from contextlib import nullcontext
+from typing import ContextManager
+
+import torch
+
+# ``nvtx_range`` lives in the main package and wraps the standalone ``nvtx``
+# package; import defensively so a missing dependency degrades to a no-op range
+# rather than failing the benchmark.
+try:
+    from tensorrt_llm._utils import nvtx_range as _nvtx_range
+except Exception:  # pragma: no cover - defensive
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)